README.rst

```diff
@@ -12,16 +12,27 @@ therefore misses many convenience features. Instead, these convenience methods
 have been split out into a layer written in PHP, the `MongoDB Library`_.
 Using this library should be your preferred way of interacting with MongoDB.
 
-Please note that the new HHVM and PHP drivers implement a different API from
-the legacy driver at http://pecl.php.net/package/mongo, and hence, you will
-find it hard for other libraries like `Doctrine MongoDB's ODM`_ will not work
-with the new drivers.
+Please note that the new HHVM and PHP drivers implement a **different API**
+from the legacy driver at http://pecl.php.net/package/mongo, and hence, you
+will find it hard for other libraries like `Doctrine MongoDB's ODM`_ will not
+work with the new drivers.
```

Comment:
Sentence structure seems incorrect. "therefore, existing libraries that use the legacy driver (e.g. `Doctrine MongoDB's ODM`_) will not work with the new drivers". If that sounds strong, you may want to add something to say that they may work after some upgrading.